docs/modules/hdfs/examples/getting_started/hdfs.yaml

```diff
@@ -11,7 +11,7 @@ spec:
     dfsReplication: 1
   nameNodes:
     config:
-      listenerClass: external-stable # This exposes your Stacklet outside of Kubernetes. Remove this configuration if this is not desired
+      listenerClass: external-unstable # This exposes your Stacklet outside of Kubernetes. Remove this configuration if this is not desired
```

Comment:
Sorry for the question, but what was the motivation behind this change?
According to https://github.com/stackabletech/decisions/issues/7 we would like to use stable here.
Is it because it doesn't work out of the box on kind? In that case the new stackablectl feature or helm feature should help, doesn't it?

Reply:
I'm working on releasing stackablectl ASAP to address this

Reply:
Release PR: https://github.com/stackabletech/stackable-cockpit/pull/415

Reply:
Ah, thanks, didn't see that until after I'd commented :)

Reply:
This now works with presets - see 317a6d1